.github/CODEOWNERS

```
@@ -27,7 +27,7 @@
 /src/azure-cli/azure/cli/command_modules/appservice/ @panchagnula @jsntcy @zhoxing-ms @yanzhudd
 /src/azure-cli/azure/cli/command_modules/aro/ @bennerv @hawkowl @jewzaam @rogbas
 /src/azure-cli/azure/cli/command_modules/backup/ @dragonfly91 @zhoxing-ms @akshayneema @jsntcy @yanzhudd
-/src/azure-cli/azure/cli/command_modules/batch/ @bgklein @gingi @dpwatrous @paterasMSFT @zhoxing-ms @jsntcy @yanzhudd
+/src/azure-cli/azure/cli/command_modules/batch/ @cRui861 @wanghoppe @dpwatrous @wiboris 
```

Please do not remove our CLI team members as code owners, otherwise we will not be able to merge PR which dose not require your review *(such as: Fix typo in help message or bump version)*
```suggestion
/src/azure-cli/azure/cli/command_modules/batch/ @cRui861 @wanghoppe @dpwatrous @wiboris @zhoxing-ms @jsntcy @yanzhudd
```